src/components/VmDetails/cards/NicsCard/NicListItem.js

```diff
@@ -40,19 +40,17 @@ const NicListItem = ({ idPrefix, nic, vmStatus, vnicProfileList, isEditing, onEd
         </span>
       </div>
       <Grid>
-        <Row>
+        <Row className={style['nics-row']}>
           <Col cols={4} className={style['ip4-container']} id={`${idPrefix}-ipv4`}>
             { nic.ipv4.length > 0 &&
-              <div>
-                {nic.ipv4.map(ip4 => <div key={`${nic.id}-${ip4}`}>{msg.nicIP4()}: {ip4}</div>)}
-              </div>
+              nic.ipv4.map((ip4, index) => <FieldValue className={style['ip']} tooltip={ip4} key={`${nic.id}-${ip4}-${index}`}
```

Comment:
This line is getting a bit long.  We usually brake long lines by putting attributes on their own lines...
```jsx
              nic.ipv4.map((ip4, index) =>
                <FieldValue
                  className={style['ip']}
                  tooltip={ip4} 
                  key={`${nic.id}-${ip4}-${index}`}
                  id={`${idPrefix}-ipv4-${index}`}
                >
                  {msg.nicIP4()}: {ip4}
                </FieldValue>
              )
```